docs/control-structures.rst

```diff
@@ -233,6 +233,10 @@ creation-dependencies are not possible.
     contract C {
         D d = new D(4); // will be executed as part of C's constructor
 
+        function C() payable {
```

Comment:
Thanks! Perhaps it would be more flexible and also easier to understand to add a payable fallback function?

Reply:
I'm not sure I follow. Why does the fallback function have to be payable if we have a payable constructor?

Reply:
If the constructor is payable, you can only send ether at the point of construction. If you have a payable fallback function, you can send ether any time. The second method is also more convenient from a syntax and user interface perspective.

Reply:
Got it! :) 
Pushed another commit.